cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -317,8 +322,8 @@
     "RAIDOptions": {
       "Description": "Comma Separated List of RAID related options, 8 parameters in total, [shared_dir,raid_type,num_of_raid_volumes,volume_type,volume_size,volume_iops,encrypted,ebs_kms_key_id]",
```

Comment:
We probably need to update this, if we are adding a parameter and chaining 9 in total now

Reply:
Fixed